src/Form/Type/Filter/DateTimeType.php

```diff
@@ -109,7 +109,7 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'field_type' => FormDateTimeType::class,
-            'field_options' => ['date_format' => 'yyyy-MM-dd'],
+            'field_options' => ['date_format' => FormDateTimeType::HTML5_FORMAT],
```

Comment:
This is `"yyyy-MM-dd'T'HH:mm:ss"`.

Why not using `DateType::HTML5_FORMAT` which is `yyyy-MM-dd` ?

Reply:
>Why not using DateType::HTML5_FORMAT which is yyyy-MM-dd ?

Because at first sight, this type being `DateTimeType` and not `DateType`, my criteria (without any analysis yet) makes me think that maybe the proper format could be one including the time part. That's the reason about the entries in the **To do** section.